bin/.gitignore

```diff
@@ -0,0 +1,22 @@
+classes/
+ext/signed/
+ext/cachedir/
+lib/
+.*.swp
+*.class
+*.pyc
+/browser/
+/build.xml
+/createKeyStore
+/deploy.sh
```

Comment:
this file is part of the repository, why do you propose to ignore it ?

Reply:
I haven't changed gitignore by hand from the origiinal code I branched off. I don't know why it became chaged meanwhile. I wil change it back to original as in master. My guess is that it has to do something with project being repeatedly opened in Eclipse.